commands/pull.js

```diff
@@ -138,32 +149,39 @@ const run = co.wrap(function * (sourceIn, targetIn) {
 function * push (context, heroku) {
   const fetcher = require('../lib/fetcher')(heroku)
   const {app, args} = context
+  const flags = context.flags
+  const exclusions = parseExclusions(flags['exclude'])
 
   const source = parseURL(args.source)
   const target = yield fetcher.database(app, args.target)
 
   cli.log(`heroku-cli: Pushing ${cli.color.cyan(args.source)} ---> ${cli.color.addon(target.attachment.addon.name)}`)
-  yield run(source, target)
+  yield run(source, target, exclusions)
   cli.log('heroku-cli: Pushing complete.')
 }
 
 function * pull (context, heroku) {
   const fetcher = require('../lib/fetcher')(heroku)
   const {app, args} = context
+  const flags = context.flags
+  const exclusions = parseExclusions(flags['exclude'])
 
   const source = yield fetcher.database(app, args.source)
   const target = parseURL(args.target)
 
   cli.log(`heroku-cli: Pulling ${cli.color.addon(source.attachment.addon.name)} ---> ${cli.color.cyan(args.target)}`)
-  yield run(source, target)
+  yield run(source, target, exclusions)
   cli.log('heroku-cli: Pulling complete.')
 }
 
 let cmd = {
   topic: 'pg',
   needsApp: true,
   needsAuth: true,
-  args: [{name: 'source'}, {name: 'target'}]
+  args: [{name: 'source'}, {name: 'target'}],
+  flags: [
+    {name: 'exclude', char: 'e', hasValue: true, description: 'tables for which data should be excluded (use \';\' to split multiple names)'}
```

Comment:
What do you think about copying the `pg_dump` flag name here (`--exclude-table-data`) and dropping the short flag (since `pg_dump` doesn't have one for this option) for consistency?

Reply:
Yes I've been hesitating on this one, and have chosen a shorter version for simplicity ; but if you intend to give more options later then it's probably better to stick to pg_dump's normalization. It will make it feel more natural for people used to pg-dump, and will bring clarity to what the option really does!